Library/ViewModels/SurveyResponseViewModel.swift

```diff
@@ -164,6 +164,10 @@ private func isSurvey(request: URLRequest) -> Bool {
   return true
 }
 
-private func isStripeElement(_ request: URLRequest) -> Bool {
-  return request.url?.host == "js.stripe.com"
+// Returns true if the url host is of the form *.stripe.com or *.stripe.network.
+private func isStripeRequest(_ request: URLRequest) -> Bool {
+  guard let host = request.url?.host?.lowercased() else { return false }
+  let components = host.split(separator: ".")
+  return components.count == 3 && components[1] == "stripe" &&
+    (components[2] == "com" || components[2] == "network")
```

Comment:
_not merge blocking:_ I'm not sure if we have other methods that check for a URL host like this, but we could consider setting up an enum to check against here to avoid hardcoding the strings and make it easier to verify other URLs in the future.

This could be too specific of a use case for that, though, so your call!

Reply:
a unit test for this might also be helpful

Reply:
I don't think it's worth setting up constants for this, to be honest. We never handle third party hosts outside of this webview, and I don't expect us to do so in the future either (unless we decide to add more webviews 😭 ). I'll add tests, though! Good call!